srcpkgs/cryfs/template

```diff
@@ -1,22 +1,23 @@
 # Template file for 'cryfs'
 pkgname=cryfs
-version=0.10.2
-revision=5
+version=0.11.2
+revision=1
 create_wrksrc=yes
 build_style=cmake
-configure_args="-DCRYFS_UPDATE_CHECKS=off
- -DBoost_USE_STATIC_LIBS=OFF -DCMAKE_CXX_STANDARD_LIBRARIES=-lfmt"
+configure_args="-DCRYFS_UPDATE_CHECKS=off -DBoost_USE_STATIC_LIBS=OFF 
+ -DCMAKE_CXX_STANDARD_LIBRARIES=-lfmt
+ -DDEPENDENCY_CONFIG=../cmake-utils/DependenciesFromLocalSystem.cmake"
 hostmakedepends="pkg-config python3"
 makedepends="boost-devel fuse-devel libcurl-devel libgomp-devel crypto++-devel
- spdlog"
+ spdlog range-v3"
```

Comment:
IIRC range-v3 can be in `hostmakedepends`, because it’s only used for code generation.

See: https://github.com/JamiKettunen/void-packages/pull/1/commits/ffb2da4ad9046b31af3d4b599173287351b8cc3f

Reply:
Nope